tool_share/arm_mounted_tray_V1/README.md

```diff
@@ -0,0 +1,29 @@
+![image](../../images/banner.png)
+
+# Arm-Mounted Tray
+
+**Created by**: Hello Robot Inc
+
+## Overview
+
+<img src="./images/bringing_condiments.jpg" alt="Stretch carrying condiments" height="400" />
+
+This tray conveniently clips onto Stretch's first arm link to provide a surface for carrying objects, including two hooks in the front and a cup holder to assist a variety of tasks. 
+
+## Parts List
+
+| Item | Qty | Vendor           |
+| ------------- |:-------------:| -----: |
+| [Tray](CAD/stretch_tray.STL) | 1 | PLA 3D Printer |
+| [Arm Clips](CAD/tray_clip.STL) | 2 | PLA 3D Printer |
```

Comment:
Looks like these STLs are missing from the CAD folder.

Reply:
Oops, thanks for catching this! I updated the files in the CAD folder.